### DISCUSSION TITLE
重复：画面加速（无垂直同步）

### DISCUSSION THREAD

**Discussion by anpoz** [hooke007/mpv_PlayKit#238] [回收]
下载的是[mpv-lazy-2022EX](https://github.com/hooke007/MPV_lazy/releases/tag/20221112)
标准版+hook的配置，默认打开什么视频都是这个样子，是不是要改什么设置？
![动画](https://user-images.githubusercontent.com/2825342/204285597-9304f047-10d0-4752-91c0-9219057d5bd9.gif)
![MPV-00_04_11 001-N0001](https://user-images.githubusercontent.com/2825342/204285815-bcb3aa31-558f-4e98-a1c1-290857a24214.png)

**Comment by hooke007** [comment]
看wiki的faq
你的显卡驱动设置有问题

https://github.com/hooke007/MPV_lazy/discussions/138
